src/Calculator/Controls/MathRichEditBox.cpp

```diff
@@ -16,6 +16,19 @@ using namespace Windows::Foundation::Collections;
 
 DEPENDENCY_PROPERTY_INITIALIZATION(MathRichEditBox, MathText);
 
+// TODO remove when Windows 10 version 2004 SDK is adopted
```

Comment:
Is this the right version number? Should it be 20004?

Reply:
"2004" is the product name that a user would see in the Settings app (April, 20**20**). Final build number not yet known.